coalib/bearlib/abstractions/Linter.py

```diff
@@ -467,6 +486,38 @@ def _create_config(cls, filename, file, **kwargs):
                     yield config_file
 
         def run(self, filename, file, **kwargs):
+            printer = ConsolePrinter()
+            log_printer = LogPrinter(NullPrinter())
+            sections, _ = load_configuration(None, log_printer)
+            local_bears, global_bears = collect_all_bears_from_sections(
+                sections, log_printer)
+            bears = inverse_dicts(local_bears, global_bears)
+            ext = os.path.splitext(filename)[1]
+            lang_map = {"all": "All"}
+            lang_map[exts[ext].lower()] = exts[ext]
+            relevant_bears = {"All": []}
+            relevant_bears[exts[ext]] = []
+            for bear in bears:
+                if isinstance(bear.LANGUAGES, tuple):
+                    bear_languages = [lang.lower() for lang in bear.LANGUAGES]
+                else:
+                    bear_languages = [bear.LANGUAGES.lower()]
```

Comment:
just use the `bear.supported_languages` property which is guaranteed to be a tuple